data/expression/script/gocc/ast/function.go

```diff
@@ -20,7 +20,7 @@ func NewFuncExpr(name interface{}, args interface{}) (Expr, error) {
 
 	f := function.Get(funcName)
 	if f == nil {
-		return nil, fmt.Errorf("function '%s' not registered", funcName)
+		return nil, fmt.Errorf("incompatible app engine, the app using the function [%s] which is not installed", funcName)
```

Comment:
how about
```
unable to evaluate expression, function [%s] is not installed
```

Reply:
@mellistibco what are your thoughts on this error message?

Reply:
I don’t like the idea of saying its an incompatible app engine. That may be a bit confusing. I’d prefer to go with something like `unable to evaluate/parse expression because function blah is not installed`